app.rb

```diff
@@ -0,0 +1,121 @@
+require_relative './book'
+require_relative './teacher'
+require_relative './student'
+require_relative './person'
+require_relative './rental'
+
+class App
+  def initialize()
+    @book_list = []
+    @rentals = []
+    @person = []
+  end
+
+  def menu
```

Comment:
- [ ] [OPTIONAL] Kindly add MIT license to your project. When MIT license link clicked on readme. It's showing broken link.